tests/research/test_b2_05_prereg_freeze.py

```
@@ -0,0 +1,624 @@
+        "impact/absorption adds stable\n> information about subsequent "
+        "directional behavior."
+    ) in text
+    # This unit must not touch the immutable inventory file at all. Pin the
+    # exact accepted-base Git blob SHA of the file's bytes rather than
+    # diffing against origin/main -- a shallow/merge-ref CI checkout does
+    # not guarantee an origin/main ref exists, but `git hash-object` works
+    # offline against whatever bytes are on disk in this checkout.
+    import subprocess
+
+    hashed = subprocess.run(
+        ["git", "hash-object", "docs/research/V2_FORMULATION_INVENTORY.md"],
```

Ruff (0.16.3)

[error] Starting a process with a partial executable path

(S607)